src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/template.json

```diff
@@ -42,6 +52,12 @@
       "datatype": "bool",
       "description": "If specified, skips the automatic restore of the project on create.",
       "defaultValue": "false"
+    },
+    "NoPagesAndViews": {
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false",
```

Comment:
I thought we were defaulting to portable components class library. If not, command-line users would have to figure out a non-obvious syntax for passing a value for `NoPagesAndViews`.

Reply:
Update: I see the default was already discussed, and we are defaulting to the MVC case for now.

Reply:
Yep, it needs to be -npv for now, I don't know if you can do something like `-npv:true` `-npv:false` or similar